src/common/progressBar.js

```diff
@@ -0,0 +1,68 @@
+/* @flow strict-local */
```

Comment:
You should delete this file, and switch to using the version in `package.json`

Reply:
I created a seperate file for progress-bar because we will have to use selector in every file(screen) to get loading status and override default props of Bar [see here](https://github.com/oblador/react-native-progress) as it has defalut `borderWidth=1, borderRadius=4, height=6, indeterminate` etc. and each time all these have to write again and again. According to user interface I used `borderWidth=0,radius=0, height=3` etc, you can see in common/progressBar.js I defined default props and overridring props as our need and also have to contain it in View to provide positional style, as Bar from react-native-progress doesn't have style props to accept. And the last, by doing all these we need to write only <ProgressBar /> in the screen and that's it. This is the same way as loadingBanner has implemented.

Reply:
from loadingBanner i am referring to connecting-spinner